MAUI/Pull-to-Refresh/pulltorefresh-events.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: PullToRefresh Events and Commands in .NET MAUI PullToRefresh control | Syncfusion
```

Comment:
Events and Commands in .NET MAUI PullToRefresh control | Syncfusion

Reply:
changed